crates/graphql/src/schema/objects/wallet.rs

```diff
@@ -430,17 +430,22 @@ impl Wallet {
     ) -> FieldResult<U64> {
         let db_conn = ctx.shared.db.get()?;
 
-        let wallet_total_rewards = wallet_total_rewards::table
+        let wallet_total_reward = wallet_total_rewards::table
             .select(wallet_total_rewards::total_reward)
             .filter(
                 wallet_total_rewards::reward_center_address
                     .eq(&reward_center)
                     .and(wallet_total_rewards::wallet_address.eq(&self.address)),
             )
             .first::<BigDecimal>(&db_conn)
-            .context("Failed to load total rewards")?;
+            .context("Failed to load total rewards");
 
-        Ok(wallet_total_rewards.try_into().unwrap_or_default())
+        let wallet_total_reward = match wallet_total_reward {
```

Comment:
You can mark the result as optional() and the use unwrap default to write the zero.

Reply:
And why big decimal isn't this i64? Should be no decimals.